service/build.gradle

```
@@ -149,13 +152,20 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop', version: versions.springBoot
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-json', version: versions.springBoot
 
+  compile 'org.json:json:20171018'
+
+  compile group: 'com.google.guava', name: 'guava', version: '23.5-jre'
```

Please use most recent version.

```suggestion
  compile group: 'com.google.guava', name: 'guava', version: '27.0-jre'
```